README.adoc

```diff
@@ -11,6 +11,79 @@ This repository follows the monorepo pattern and includes the following projects
 - **verifier**: A command line application to audit ballots and verify the election results.
 - **voting_schemes**: Implementation of different cryptographic voting schemes to be used to run the elections. Currently, it includes the Dummy scheme, used only for testing purposes, and the ElectionGuard scheme, based on the Python implementation of the protocol.
 
-## License
+== Getting started with this repository
+
+- Clone this repository with submodules `git clone --recurse-submodules -j8 https://github.com/decidim/decidim-bulletin-board.git`
+- Or clone the submodules separately after the initial clone by running `git submodule update --init --recursive` in the repository folder
+- Use one of the development methods below
+
+=== Devcontainer
+
+This repository ships with a dev container to get you up and running quickly. This make is easy to get the development environment running without having to install all the dependencies.
+
+Read more about https://code.visualstudio.com/docs/remote/containers[devcontainers on Visual Studio Code documentation].
+
+=== Manual development environment
+
+==== Install Ruby and Node
+
+Assuming you already know how to if you are already working with Decidim.
+
+==== Install Python
+
+This guide is assuming that you don't have any python development tooling installed on your home folder. If so, we recommend that you start with a clean state just to be sure, for instance, deleting .local/bin/pip with rm $[HOME}/.local/bin/pip
+
+You can use e.g. https://github.com/pyenv/pyenv[pyenv] to install the correct version.
```

Comment:
When running this on a clean LXC environment, I saw some missing dependencies. We could add them (at least for Ubuntu, given that we're assuming instructions for that OS), or link to the official documentation: 
```suggestion
You can use e.g. https://github.com/pyenv/pyenv[pyenv] to install the correct version. You should fulfill the dependencies on you Operating System, see the https://github.com/pyenv/pyenv/wiki#suggested-build-environment[pyenv's suggested build environment].
```